prow/config/config.yaml

```diff
@@ -36,6 +36,7 @@ tide:
     Nordix/metal3-clusterapi-docs: rebase
   queries:
     - repos:
+        - metal3-io/.github
```

Comment:
this is not correct actually (should be metal3-io/metal3-io.github.io), besides it is already there (line 49)

Reply:
No, `.github` is special repository on the organization level, it is not the same as the book :)

Reply:
its different @furkatgofurov7 , we have added also .github repo

Reply:
Both repos exist actually.
- https://github.com/metal3-io/metal3-io.github.io
- https://github.com/metal3-io/.github

Reply:
Thanks @kashifest @lentzi90